apps/dev/src/formatters/formatters-demo.component.ts

```diff
@@ -16,6 +16,7 @@
 
 import { Component } from '@angular/core';
 
+/** @breaking-change - Removed with version 7.0.0 */
```

Comment:
This is in the dev app, - no breaking change annotation needed. And we will still have other formatters that we need to demo. 
Remove this comment please :)